app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -1694,6 +1695,11 @@ public boolean isWebXRPresenting() {
         return mIsPresentingImmersive;
     }
 
+    @Override
+    public boolean isPassthroughEnabled() {
+        return mIsPassthroughEnabled;
+    }
```

Comment:
I don't think this is the right place for this, I think a better place is to store it in the settings

Reply:
Umm, I thought about it. However, I wasn't sure we wanted a persistent storage for this state; do we really want to launch Wolvic under passtrhough mode if it was enabled before closing it ?

Reply:
That's a good question. We do save the current state of the browser, sessions, windows, history, etc... when exiting so it looks like passthrough state could be saved too?

Reply:
ok, we can give it a try then,